django/chat/static/chat/scripts.js

```diff
@@ -5,9 +5,9 @@ const copyCodeButtonHTML = `<button type="button" onclick="copyCode(this)"
 class="btn btn-link m-0 p-0 text-muted copy-message-button copy-button"
 title="Copy"><i class="bi bi-clipboard"></i><i class="bi bi-clipboard-fill"></i></button>`;
 
-function scrollToBottom(smooth = true) {
+function scrollToBottom(smooth = true, force = false) {
   resizePromptContainer();
-  if (preventAutoScrolling) {
+  if (preventAutoScrolling && !force) {
```

Comment:
Force the chat window to scroll to the bottom when a new user message is submitted.